### ISSUE TITLE
The ads are muted but the playlist stops, and the volume never goes up to 100% again

### ISSUE THREAD

**Issue by ncoletto** [ArtemGordinsky/Spotifree#93]
For example, the ads are muted but the playlist stops, and the volume never goes up to 100% again. In this scenario, I have to close and open Spotify again to solve this. 

This situation happens randomly. Some ads are muted as they should, some are not, and some of them causes the playlist to stop and the volume remains at 0%. 

I write because I want you to be aware of this and maybe some more experienced programmer can make an update. Or give some advice to solve. Thanks!

**Comment by sippsolutions** [comment]
Can confirm

**Comment by mirkofetter** [comment]
Same issue

**Comment by mirkofetter** [comment]
Same issue here.

**Comment by Enellis** [comment]
It would be very helpful if you would say what Spotify and Spotifree version you are using.

**Comment by sippsolutions** [comment]
Both the latest versions.

**Comment by Enellis** [comment]
What version number? The latest release on Github is 1.6.5 but it could also be 2.0.

**Comment by mirkofetter** [comment]
I use 1.6.5 with OS X 10.11.6  and  10.12.3

**Comment by Enellis** [comment]
Okay. Could you try the newest development build and tell me whether it is fixed for you?
https://www.dropbox.com/s/pcxsefdth8h2hdk/Spotifree2dev.app.zip?dl=0

**Comment by ncoletto** [comment]
Hey Enellis! I can confirm that by downloading your 2.0dev version of Spotifree, now this is working better! So far I just tested it yesterday and today. But sadly more adds than before are skipped ok, but from time to time one fails, stop the playlist and keep the volume up to 0%. (My specs Hackintosh OSX 10.11.2, Spotify 1.0.59.395, Spotifree 2.0dev). Thanks!! At least I can help with testing and reporting back..

**Comment by Enellis** [comment]
Thank you. I really appreciate your help!
I built a new version. Can you download it again and tell me if it works now?

**Comment by Enellis** [comment]
Also, does the menu bar icon of Spotifree change to the muting icon when this bug happens?

**Comment by mirkofetter** [comment]
Had the same issues with the old dev, know trying the new one.

**Comment by ncoletto** [comment]
Hey Enellis! this is my reporting using the new dev file yesterday. First of all answer your question. Yes Spotifree change its icon in the menu bar when mutes the add. So far, for me is working better. I only had a couple of issues: At least one add was not muted, so I heard it. Most of them are muted correctly. And happened to me twice that when I change the band, I clic on the first song of the first album you see on the list (latest album), Spotifree lowers the volume. So I only put the volume up and thats it. 
Thats the only 2 things that happened. Otherwise, it makes its job much better than before...

**Comment by Enellis** [comment]
@ncoletto Did the ad which was not getting muted occur after putting the volume up manually because Spotifree falsely lowered the volume?

**Comment by ncoletto** [comment]
The answer is almost a yes. I am not sure, because that happened only once that time. All this days, I continue to test and use it and I never hear an add again. So the only, lets say, bug that remains is the one that puts the volume down when you clic on a new album.. and happens some times random, not all the times, or not in the same album..

**Comment by ncoletto** [comment]
Yes! I can confirm what you said. Just now happened this this sequence: I click on a song. (the volume lowered itself down to 0%). Then I put the volume up to 100%. After that song, an add started and I heard it, so this one was not muted..

**Comment by Enellis** [comment]
Okay great. Now I know what happens there. That means the last problem I have to fix is the falsely lowering of the volume.

So, is it just happening if you click on a new song maually? Is it happening to local files only?

**Comment by ncoletto** [comment]
Exactly. In my case, yes, I only click manually on internet stuff, not using local files.
